### ISSUE TITLE
kmail.profile broken due to wildcard

### ISSUE THREAD

**Issue by davidebeatrici** [netblue30/firejail#4416]
```
Error: "${HOME}/.cache/akonadi*" is an invalid filename: rejected character: "*"
Error: "${HOME}/.config/akonadi*" is an invalid filename: rejected character: "*"
Error: "${HOME}/.local/share/akonadi*" is an invalid filename: rejected character: "*"
Error: "/tmp/akonadi-*" is an invalid filename: rejected character: "*"
```

Looks like the issue was originally introduced in 5532fbdb9749c5333ac03152f8c94fd364182d32.

**Comment by rusty-snake** [comment]
Duplicate of #3979

**Comment by rusty-snake** [comment]
Fixed in bb9107e2ae25ad7ce38ba4f2eb898d384aa20d19.

What's you firejail version?

> Looks like the issue was originally introduced in ecbf5dd.

I don't have any issues with a `*` in a `noblacklist` command (for `mkdir` is invalid).

**Comment by davidebeatrici** [comment]
> What's you firejail version?

0.9.65

> Duplicate of #3979

> Fixed in bb9107e.

> > Looks like the issue was originally introduced in [ecbf5dd](https://github.com/netblue30/firejail/commit/ecbf5ddb450ba0ad86d9a892e9bc14d52ad86fa4).
> 
> I don't have any issues with a `*` in a `noblacklist` command (for `mkdir` it's invalid).

Sorry, I realized later that only the `mkdir` directives cause the issue. As a result I didn't notice it's already fixed in `master`.
